client/state/action-types.js

```diff
@@ -330,6 +330,11 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
 export const HISTORY_REPLACE = 'HISTORY_REPLACE';
+export const HOSTING_CREATE_SFTP_USER = 'HOSTING_CREATE_SFTP_USER';
+export const HOSTING_RECEIVE_SFTP_USER = 'HOSTING_RECEIVE_SFTP_USER';
+export const HOSTING_RECEIVE_SFTP_USER_ERROR = 'HOSTING_RECEIVE_SFTP_USER_ERROR';
```

Comment:
We can drop this one too, making an optimistic update, then dispatching `HOSTING_SFTP_USER_CREATE` or `HOSTING_SFTP_USER_REMOVE` would read better.